apps/public-reference/pages/_app.jsx

```diff
@@ -98,6 +106,7 @@ class MyApp extends App {
     return (
       <AppSubmissionContext.Provider
         value={{
+          conductor: this.state.conductor,
```

Comment:
I was just looking at the logic here for `syncApplication` and `syncListing` - it seems a bit problematic to me to have the "split state" between exposing those objects directly via the context, but then also having the conductor keep its own internal references to them. Right now if `syncApplication` or `syncListing` are used to change the global `application`/`listing` values to different objects, the conductor doesn't get updated - it will just keep the old values.

To me, it seems like the cleanest way to do this is to _only_ expose the conductor, and just let it be the single interface for dealing with these variables, but if you wanted to, you could also modify the `syncApplication` and `syncListing` functions to update the conductor as well as the state variables.

Worth noting that If you _do_ end up consolidating everything into the conductor, you'll basically have created a ContextProvider though - it'd be a quick & easy refactor at that point to update it so that rather than creating a context and giving it a reference to the conductor, you have the conductor be a React component that provides the context to the tree. I think this would result in something that's a bit a easier to scope, but it's optional ;-).

Reply:
@bencpeters Yeah, I thought about that too, but see my comment above…they should be used as close to never as possible since we're modifying the application date in-place as we go, and the listing is essentially read-only.

Reply:
That makes sense, but to me the fact that they’re not even really used at present means we should either fix or remove them - right now it’s potentially dangerous code that isn’t really being tested, but could lead to inconsistent state if used down the line.

In this case “fixing” could be as easy as modifying the `sync` functions to also update `application`/`listing` on the conductor too...

Reply:
Truth — for now I'd say let's put a pin in it…if we end up "merging" application and listing data models together in some fashion ala https://github.com/bloom-housing/bloom/issues/453, we might not even need to track both separately.